src/App.tsx

```diff
@@ -15,36 +17,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABETIC = 'alphabet',
+  LENGTH = 'length',
```

Comment:
[CODE STYLE] - Enum constant names usually start with capital letter. Values can have any case that's necessary.
![image](https://github.com/mate-academy/react_list-of-goods-hooks/assets/63243136/036dc54e-d0b5-40e0-bd55-4e7a3cd2bfbc)